core/src/main/java/org/elasticsearch/search/aggregations/AggregationBuilder.java

```diff
@@ -84,9 +83,10 @@ public String getName() {
     }
 
     /**
-     * Internal: Registers sub-factories with this factory. The sub-factory will be
-     * responsible for the creation of sub-aggregators under the aggregator
-     * created by this factory. This is only for use by {@link AggregatorFactories#parseAggregators(QueryParseContext)}.
+     * Internal: Registers sub-factories with this factory. The sub-factory will
+     * be responsible for the creation of sub-aggregators under the aggregator
+     * created by this factory. This is only for use by
+     * {@link AggregatorFactories#parseAggregators(org.elasticsearch.common.xcontent.XContentParser)}.
```

Comment:
I think it is ok to import `XContentParser` to make the javadoc shorter if you like.